draft-thomson-httpbis-http2bis.xml

```diff
@@ -1167,231 +1162,77 @@
-            parent of other streams.  This allows for the creation of a grouping node in the
-            dependency tree, which enables more flexible expressions of priority.  Idle streams
-            begin with a <xref target="pri-default">default priority</xref>.
+            This update to HTTP/2 deprecates the priority signaling defined in <xref
+            target="RFC7540">RFC 7540</xref>.  The bulk of the text related to priority signals is
+            not included in this document.  The description of frame fields and some of the
+            mandatory handling is retained to ensure that implementations of this document remain
+            interoperable with implementations that use the priority signaling described in RFC
+            7540.
           </t>
           <t>
-            The retention of priority information for streams that are not counted toward the limit
-            set by <xref target="SETTINGS_MAX_CONCURRENT_STREAMS" format="none">SETTINGS_MAX_CONCURRENT_STREAMS</xref> could create a large state burden
-            for an endpoint.  Therefore, the amount of prioritization state that is retained MAY be
-            limited.
+            A thorough description of the RFC 7540 priority scheme remains in <xref target="RFC7540"
+            section="5.3"/>.
           </t>
           <t>
-            The amount of additional state an endpoint maintains for prioritization could be
-            dependent on load; under high load, prioritization state can be discarded to limit
-            resource commitments.  In extreme cases, an endpoint could even discard prioritization
-            state for active or reserved streams. If a limit is applied, endpoints SHOULD maintain
-            state for at least as many streams as allowed by their setting for
-            <xref target="SETTINGS_MAX_CONCURRENT_STREAMS" format="none">SETTINGS_MAX_CONCURRENT_STREAMS</xref>.  Implementations SHOULD also attempt to
-            retain state for streams that are in active use in the priority tree.
+            Signaling priority information could still be necessary to attain good performance.
+            Where signaling priority information is important, endpoints are encouraged to use an
+            alternative scheme, such as <xref target="I-D.ietf-httpbis-priority"/>.
           </t>
           <t>
-            If it has retained enough state to do so, an endpoint receiving a
-            <xref target="PRIORITY" format="none">PRIORITY</xref> frame that changes the priority of a
-            closed stream SHOULD alter the dependencies of the streams that depend on it.
+            Though the priority signaling from RFC 7540 was not widely adopted, the information it
+            provides can still be useful in the absence of better information.  Endpoints that
+            receive <xref target="HEADERS" format="none">HEADERS</xref> or <xref target="PRIORITY"
```

Comment:
Seeing as the extensible priorities draft also utilizes HTTP headers to communicate its signals, I wonder if this mention of HEADERS frames is too ambiguous for uninitiated readers here? Maybe something like "HEADERS (as defined in RFC 7540)" ?

Reply:
I've tweaked this to say "if you receive priority signals from HEADERS or PRIORITY frames", which should be clear enough.